BrowserKit/Sources/WebEngine/WKWebview/WKEngineSession.swift

```diff
@@ -354,6 +354,7 @@ class WKEngineSession: NSObject,
         contextMenuConfigurationForElement elementInfo: WKContextMenuElementInfo,
         completionHandler: @escaping (UIContextMenuConfiguration?) -> Void
     ) {
+        completionHandler(delegate?.onProvideContextualMenu(linkURL: elementInfo.linkURL))
         // TODO: FXIOS-8246 - Handle context menu in WebEngine (epic part 3)
```

Comment:
Seems FXIOS-8246 is in fact a duplicate 🤔 Oups. Let me close that one and we can remove this comment
```suggestion
```